Src/Base/AMReX_GpuQualifiers.H

```diff
@@ -30,6 +31,7 @@
 #define AMREX_GPU_CONSTANT
 #define AMREX_GPU_MANAGED
 #define AMREX_GPU_DEVICE_MANAGED
+#define AMREX_GPU_EXTERNAL SYCL_EXTERNAL
```

Comment:
why this?

Reply:
In sycl, to use an external function on device, one must mark it SYCL_EXTERNAL.